src/web/lib/components/App/index.js

```diff
@@ -3,12 +3,13 @@ import { connect } from 'react-redux';
 
 import { actions, selectors } from '../../../../lib/store';
 
+import AppHeader from '../AppHeader';
 import AppBackground from '../AppBackground';
 import BrowserPreview from '../BrowserPreview';
 import ThemeColorsEditor from '../ThemeColorsEditor';
 import PresetThemeSelector from '../PresetThemeSelector';
 import ThemeBackgroundPicker from '../ThemeBackgroundPicker';
-import ExtensionInstallButton from '../ExtensionInstallButton';
+import Banner from '../Banner';
```

Comment:
I made this more generic since in think we could use the same component for the install firefox cta